tiledb/sm/filter/filter.h

```diff
@@ -86,7 +103,7 @@ class Filter {
    */
   Filter* clone(Datatype data_type) const;
 
-  /** Dumps the filter details in ASCII format in the selected output. */
+  /** Dumps the filter details in ASCII format in the selected output file. */
   virtual void dump(FILE* out) const = 0;
 
```

Comment:
This function and all of its implementations should be eliminated.

```suggestion
```

Reply:
This is a (at least) 4-year old function. Doesn't it mean that if somebody uses it, we are suddenly break his/her code?